docs: add community plugins disclaimer (#6132)
fastify/fastify: docs/Guides/Ecosystem.md

#### [Community](#community)

> > 🛈 Note:

Reviewer: I have a feeling this is not rendered correctly.

Reviewer 2: See the actual rendering 
<img width="1034" alt="fastify - Vivaldi 2025-05-20 06-24-26" src="https://github.com/user-attachments/assets/ebd55094-e855-4fb7-8e9e-8730889ee025" />

Reviewer 3: > See the actual rendering

Bloody hell, looks fine in Firefox and Edge:
![image](https://github.com/user-attachments/assets/00d97c9f-449b-45ca-8390-dddfa5482b29)

But when looking at it on Safari on iPhone it looks like your example.

Reviewer 2: Use ℹ️?

> > ⚠ Warning:

Reviewer 2: ```suggestion
> ⚠ Notice:
```

> Fastify community plugins are part of the broader community efforts,
> and we’re thankful for these contributions. However, they are not

> > maintained or reviewed by the Fastify team.

Reviewer 3: ```suggestion
> maintained by the Fastify team.
```

Most do get reviewed though when a PR is made.

> > ⚠ Notice:
> > Fastify community plugins are part of the broader community efforts,
> > and we’re thankful for these contributions. However, they are not
> > maintained by the Fastify team.
> > Use them at your own discretion.
> > If you find malicious code, please
> > [open an issue](https://github.com/fastify/fastify/issues/new/choose) or
> > submit a PR to remove the plugin.

Reviewer 3: ```suggestion
> 🛈 Note:
> Fastify community plugins are part of the broader community efforts,
> and we are thankful for these contributions. However, they are not
> maintained by the Fastify team.
> Use them at your own discretion.
> If you find malicious code, please
> [open an issue](https://github.com/fastify/fastify/issues/new/choose) or
> submit a PR to remove the plugin from the list.
```
